logicrunner/handle_calls.go

```diff
@@ -70,7 +70,7 @@ func (h *HandleCall) handleActual(
 		if err == flow.ErrCancelled {
 			return nil, err // message bus will retry on the calling side in ContractRequester
 		}
-		return nil, errors.Wrap(err, "[ handleActual ] can't play role")
+		return nil, errors.Wrap(err, "[ HandleCall ] can't play role")
```

Comment:
method named handleActual but error message changed to HandleCall, why?

Reply:
ok fixed